src/components/ButtonPanel.js

```diff
@@ -1,36 +1,37 @@
 import React from 'react';
 import Button from './Button';
+import Styles from '../styles';
 
 const ButtonPanel = () => (
-  <div>
-    <div>
-      <Button btnName="AC" />
-      <Button btnName="+/-" />
-      <Button btnName="%" />
-      <Button btnName="÷" />
+  <div style={Styles.panel}>
+    <div style={Styles.rowPanel}>
+      <Button btnName="AC" wide={false} color="#e0e0e0" />
+      <Button btnName="+/-" wide={false} color="#e0e0e0" />
+      <Button btnName="%" wide={false} color="#e0e0e0" />
+      <Button btnName="÷" wide={false} />
     </div>
-    <div>
-      <Button btnName="7" />
-      <Button btnName="8" />
-      <Button btnName="9" />
-      <Button btnName="X" />
+    <div style={Styles.rowPanel}>
+      <Button btnName="7" wide={false} color="#e0e0e0" />
+      <Button btnName="8" wide={false} color="#e0e0e0" />
+      <Button btnName="9" wide={false} color="#e0e0e0" />
+      <Button btnName="X" wide={false} />
     </div>
-    <div>
-      <Button btnName="4" />
-      <Button btnName="5" />
-      <Button btnName="6" />
-      <Button btnName="-" />
+    <div style={Styles.rowPanel}>
+      <Button btnName="4" wide={false} color="#e0e0e0" />
+      <Button btnName="5" wide={false} color="#e0e0e0" />
+      <Button btnName="6" wide={false} color="#e0e0e0" />
+      <Button btnName="-" wide={false} />
```

Comment:
[Optional]
after the above changes in `button.js`, we can now stop to repeat some code in here.
only `line 34` can now be `wide={false}`